<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CellLimitsDialog</class>
 <widget class="QDialog" name="CellLimitsDialog">
  <property name="windowModality">
   <enum>Qt::WindowModality::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>493</width>
    <height>327</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Show Multiple Copies of Cell</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Generate a slab of atoms/molecules with specified cell bounds</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="boundsGroupBox">
     <property name="title">
      <string>Bounds</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="aAxisLowerBound">
        <property name="minimum">
         <double>-99.000000000000000</double>
        </property>
        <property name="stepType">
         <enum>QAbstractSpinBox::StepType::AdaptiveDecimalStepType</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QDoubleSpinBox" name="bAxisLowerBound">
        <property name="minimum">
         <double>-99.000000000000000</double>
        </property>
        <property name="stepType">
         <enum>QAbstractSpinBox::StepType::AdaptiveDecimalStepType</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QDoubleSpinBox" name="bAxisUpperBound">
        <property name="minimum">
         <double>-99.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>1.000000000000000</double>
        </property>
        <property name="stepType">
         <enum>QAbstractSpinBox::StepType::AdaptiveDecimalStepType</enum>
        </property>
        <property name="value">
         <double>1.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="bAxisLabel">
        <property name="text">
         <string>b-axis</string>
        </property>
        <property name="buddy">
         <cstring>bAxisLowerBound</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="lowerLabel">
        <property name="text">
         <string>lower</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QDoubleSpinBox" name="aAxisUpperBound">
        <property name="minimum">
         <double>-99.000000000000000</double>
        </property>
        <property name="stepType">
         <enum>QAbstractSpinBox::StepType::AdaptiveDecimalStepType</enum>
        </property>
        <property name="value">
         <double>1.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="aAxisLabel">
        <property name="text">
         <string>a-axis</string>
        </property>
        <property name="buddy">
         <cstring>aAxisLowerBound</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="upperLabel">
        <property name="text">
         <string>upper</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="cAxisLabel">
        <property name="text">
         <string>c-axis</string>
        </property>
        <property name="buddy">
         <cstring>cAxisLowerBound</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QDoubleSpinBox" name="cAxisLowerBound">
        <property name="minimum">
         <double>-99.000000000000000</double>
        </property>
        <property name="stepType">
         <enum>QAbstractSpinBox::StepType::AdaptiveDecimalStepType</enum>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QDoubleSpinBox" name="cAxisUpperBound">
        <property name="minimum">
         <double>-99.000000000000000</double>
        </property>
        <property name="stepType">
         <enum>QAbstractSpinBox::StepType::AdaptiveDecimalStepType</enum>
        </property>
        <property name="value">
         <double>1.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="optionsGroupBox">
     <property name="title">
      <string>Other options</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="modeLabel">
        <property name="text">
         <string>Include</string>
        </property>
        <property name="buddy">
         <cstring>modeComboBox</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="modeComboBox"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>aAxisLowerBound</tabstop>
  <tabstop>aAxisUpperBound</tabstop>
  <tabstop>bAxisLowerBound</tabstop>
  <tabstop>bAxisUpperBound</tabstop>
  <tabstop>cAxisLowerBound</tabstop>
  <tabstop>cAxisUpperBound</tabstop>
  <tabstop>modeComboBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CellLimitsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>260</x>
     <y>322</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CellLimitsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>328</x>
     <y>322</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
